addons/crm/migrations/12.0.1.0/post-migration.py

```diff
@@ -29,10 +29,16 @@ def map_crm_team_dashboard_graph_model(cr):
         table='crm_team', write='sql')
 
 
+def fill_crm_lead_expected_revenue(env):
```

Comment:
No need of computing it again. As the SQL column doesn't exist, regular update process creates the column, and compute the value for it.